docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md

```diff
@@ -118,17 +117,17 @@ For more information refer to the [Advanced Test Info](AdvancedTestInfo.md) docu
 
 To install the SDK on the local machine use the following:
 
-```cmd
-cd build\x64-Debug
+```powershell
+cd build/x64-Debug
 ninja install
 ```
 
-This installs the SDK in `C:\openenclave`.
+This installs the SDK in `C:/openenclave/build/x64-Debug/install`.
```

Comment:
It would be better if we could clarify here that to install the SDK for the machine, you change the CMAKE_INSTALL_PATH to c:/openenclaveinstall or something similar. It is awkward that the SDK would be installed and used from a build directory.

Reply:
Yeah but these instructions are for contributors, right? If someone wants to install the SDK for the machine (and they're not a contributor), they'll install a package would puts it _somewhere_ for them. Contributors I think can figure out how to adjust their install path from these instructions.

Reply:
Actually, I thought I clarified that above:

> Later, using the `ninja install` command will install the SDK in `C:/openenclave/build/x64-Debug/install`. To choose a different location, change the value specified for `CMAKE_INSTALL_PATH`, but note that the samples tests will likely break.